src/libs/portfolio/helpers.ts

```diff
@@ -443,3 +445,29 @@ export const processTokens = (
     return tokens
   }, [] as TokenResult[])
 }
+
+export const isPortfolioGasTankResult = (
+  result: NetworkState['result']
+): result is PortfolioGasTankResult => {
+  return !!result && 'gasTankTokens' in result && Array.isArray(result.gasTankTokens)
+}
+
+export const shouldShowConfettiOnFirstCashback = (
```

Comment:
One last "confetti" ref. I'd name this `getIsFirstCashbackReceived` or something.